x/mongo/driver/description/topology_test.go

```diff
@@ -0,0 +1,59 @@
+package description
```

Comment:
Any new files added need a license. You can run `etc/add-license.sh x/mongo/driver/description/topology_test.go` from the top-level driver directory to add this.

Reply:
Added.